go/olc.go

```diff
@@ -167,16 +169,20 @@ func upper(b byte) byte {
 	return b
 }
 
-// stripCode strips the padding and separator characters from the code.
-func stripCode(code string) string {
-	return strings.Map(
+// StripCode strips the padding and separator characters from the code.
```

Comment:
Update the docs to say that the code is truncated if needed.

Reply:
Done.